code/03_census_and_bed_days.R

```diff
@@ -0,0 +1,868 @@
+            DelayOver2wks=sum(DelayOver2wks,na.rm=TRUE), acute=sum(acute,na.rm=TRUE),
+            gpled=sum(gpled,na.rm=TRUE),notgpled=sum(notgpled,na.rm=TRUE)) %>% 
+  ungroup()
+
+
+
+labysubreasongrouping <- datafile4 %>% 
+  group_by(year,MONTHFLAG,level,areaname,age_grp,reas1) %>% 
+  summarise(NoofPatients=sum(NoofPatients,na.rm=TRUE),Delay1to3days=sum(Delay1to3days,na.rm=TRUE),
+            Delay3to14days=sum(Delay3to14days,na.rm=TRUE),Delay2to4weeks=sum(Delay2to4weeks,na.rm=TRUE),
+            Delay4to6weeks=sum(Delay4to6weeks,na.rm=TRUE),Delay6to12weeks=sum(Delay6to12weeks,na.rm=TRUE),
+            Delay3to6months=sum(Delay3to6months,na.rm=TRUE),Delay6to12months=sum(Delay6to12months,na.rm=TRUE),
+            DelayOver12months=sum(DelayOver12months,na.rm=TRUE),DelayOver3days=sum(DelayOver3days,na.rm=TRUE),
+            DelayUnder2wks=sum(DelayUnder2wks,na.rm=TRUE),
+            DelayOver6wks=sum(DelayOver6wks,na.rm=TRUE),DelayOver4wks=sum(DelayOver4wks,na.rm=TRUE),
+            DelayOver2wks=sum(DelayOver2wks,na.rm=TRUE), acute=sum(acute,na.rm=TRUE),
+            gpled=sum(gpled,na.rm=TRUE),notgpled=sum(notgpled,na.rm=TRUE)) %>% 
+  ungroup()
+
+#write_sav(labysubreasongrouping,paste0(filepath,"la by sub reason grouping.sav")) # save out file
+
+
+### 9. Add files together ----
+
+Scot_HB_la<-bind_rows(Scotlandbymainreasongrouping,Scotlandbysubreasongrouping,HBbymainreasongrouping,HBbysubreasongrouping,
+                      labymainreasongrouping,labysubreasongrouping)
+
+
+
+#if reas1 is blank(N/A), reas1 = reas2
+
+datafile6<-Scot_HB_la%>% 
+  mutate(reas1 = as.character(reas1))%>% 
+  mutate(reas1=
+           if_else(is.na(reas1), reas2,reas1))
+
+#remove reason_grp
+datafile6<-select(datafile6,-reas2)
+
+datafile6a<-datafile6
+datafile6a$age_grp<-"All"
+ScotHBlaallage_grps<- datafile6a %>% 
+  group_by(year,MONTHFLAG,level,areaname,age_grp,reas1) %>% 
+  summarise(NoofPatients=sum(NoofPatients,na.rm=TRUE),Delay1to3days=sum(Delay1to3days,na.rm=TRUE),
+            Delay3to14days=sum(Delay3to14days,na.rm=TRUE),Delay2to4weeks=sum(Delay2to4weeks,na.rm=TRUE),
+            Delay4to6weeks=sum(Delay4to6weeks,na.rm=TRUE),Delay6to12weeks=sum(Delay6to12weeks,na.rm=TRUE),
+            Delay3to6months=sum(Delay3to6months,na.rm=TRUE),Delay6to12months=sum(Delay6to12months,na.rm=TRUE),
+            DelayOver12months=sum(DelayOver12months,na.rm=TRUE),DelayOver3days=sum(DelayOver3days,na.rm=TRUE),
+            DelayUnder2wks=sum(DelayUnder2wks,na.rm=TRUE),
+            DelayOver6wks=sum(DelayOver6wks,na.rm=TRUE),DelayOver4wks=sum(DelayOver4wks,na.rm=TRUE),
+            DelayOver2wks=sum(DelayOver2wks,na.rm=TRUE), acute=sum(acute,na.rm=TRUE),
+            gpled=sum(gpled,na.rm=TRUE),notgpled=sum(notgpled,na.rm=TRUE)) %>% 
+  ungroup()
+
+#write_sav(ScotHBlaallage_grps,paste0(filepath,"ScotHBla all age_grps.sav")) # save out file
+
+
+ScotHBlaallreasonexcHSCPatFamtotal<-bind_rows(datafile6,ScotHBlaallage_grps)
```

Comment:
See "Style Guide" and "Incremental Data-Frame Names" sections in general feedback comments.
```suggestion
ScotHBlaallreasonexcHSCPatFamtotal <- bind_rows(Scot_HB_la, ScotHBlaallage_grps)
```